<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeriesSelectDialog</class>
 <widget class="QDialog" name="SeriesSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>序列选择</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#label{
font-size: 24px;
font-family: &quot;思源黑体 CN Medium&quot;;
font-weight: 500;
color: #FFFFFF;
}
#pushButton_close{
background:transparent;
}
#widget_3{
border-top:2px solid #282C36;
border-bottom:2px solid #282C36;
}
#widget_3,#SeriesSelectDialog{
background:#1C212D;
}
QTableWidget{
gridline-color:#303439;
}
QTableWidget::item
{
border:1px solid #303439;
border-right:none;
}
QTableWidget::item:selected{
background: #1167CF;
}
QHeaderView::section:horizontal,QTableCornerButton::section{
background:#131720;
border:1px solid #303439;
}
QTableView,
QTableWidget QWidget,QTableWidget::item{
background:#212733;
}
QHeaderView::section:vertical
{
background:#212733;
border-left:none;
border-top:none;
border-right:1px solid #303439;
border-bottom:1px solid #303439;
}
QTableCornerButton::section{
border-right:1px solid #303439;
border-bottom:1px solid #303439;
border-top:none;
border-left:none;
}
QTableWidget QWidget{
border:none;
}
QHeaderView::section:checked {     /*表头分段样式*/     
background-color: #1167CF;     
}
QTableWidget
{
border: 2px solid #30363F;
border-radius: 4px 4px 0px 0px;
}
QHeaderView::section:checked {     /*表头分段样式*/     
background-color: #1167CF;     
}
QTableWidget
{
border: 2px solid #30363F;
border-radius: 4px 4px 0px 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>26</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget_series">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>19</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>34</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff></normaloff></iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>19</width>
          <height>19</height>
         </size>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="TextSize" stdset="0">
         <string>18</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>34</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff></normaloff></iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>19</width>
          <height>19</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="TextSize" stdset="0">
         <string>18</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>tableWidget_series</tabstop>
 </tabstops>
 <resources>
  <include location="qr.qrc"/>
 </resources>
 <connections/>
</ui>
